application/modules/home/views/side_ci.php

```diff
@@ -0,0 +1,5 @@
+<h3 class="box-title"><i class="fa fa-building"></i>
+<?php echo $exhibition['name']; ?>
```

Comment:
__[code_sniffer]__
* Any char before the opening PHP tag is prohibited. Please remove newline or indentation before the opening PHP tag.
* Missing file doc comment
* Any char after the closing PHP tag is prohibited. Please removes newline or spaces after the closing PHP tag.

<!-- Comment by [SideCI](https://sideci.com) -->